cypress/e2e/CMS/$HomeAndLandingPageTestCreationOfContent/index.js

```diff
@@ -0,0 +1,215 @@
+  })
+
+And('user filters results by {string} {string}',(title,option)=> {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').parent().find(`label:contains("${title}")`).parent().find("select[name='status']").select(option)
+})
+
+And('user selects {int} item from the media list',(num)=> {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-entity-browser-select-media1']").eq(num-1).check()
+})
+
+let imageSrc2;
+And('user selects {int} item from the media list',(num)=> {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-entity-browser-select-media1']").eq(num-1).check()
+})
+
+And('user remembers the name of media card item for future verification', () => {
+    cy.get('details img').then($el => {
+        imageSrc2 = $el[0].getAttribute('src')
+    })
+})
+
+/*---------- Scenario: Add List ----------*/
+And('user selects {string} item from the media list', (title) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[name='name']").type(title)
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-submit-cgov-media-browser']").click()
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find(`td:contains(${title})`).parent().find('input').click();
+})
+
+And('user clicks on {string} item button',(mediaBtn) =>{
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find(`input[value="${mediaBtn}"]`).click({force: true})
+
+})
+
+/*----------- Scenario: Add One and two column containers  -------------*/
+And('user selects {string} from main contents dropdown',(dropdown)=>{
+    cy.get(`tbody input[value="${dropdown}"]`).click({force: true})
+})
+
+And('user clicks on {string} option to {string} within {string} section',(option, containerContents,section) => {
+    cy.get(`div[id*="top-paragraph-type-title"]:contains("${section}")`).parent().parent().find(`.placeholder:contains("${containerContents}")`).parent().find(`input[value="${option}"]`).click({force:true})
+    }) 
+ 
+And('user fills out HTML Content text area with {string}',(value) =>{
+    cy.getNthIframe("iframe[title='Rich Text Editor, HTML Content field']", 1).find('p').type(value)
+})
+
+And('user enters {string} in Raw HTML Content under one column container',(value) =>{
+    cy.get("textarea[id^='edit-field-landing-contents-2-subform-field-main-contents-2-subform-field-raw-html-0-value']").type(value)
+})
+
+And('user enters {string} in Raw HTML Content under Two Column Container', (value)=>{
+    cy.get("textarea[id^='edit-field-landing-contents-3-subform-field-main-contents-0-subform-field-raw-html-0-value']").type(value)
+})
+
+And('user selects {string} from Source View dropdown',(sourceDropdown) =>{
+    cy.get("select[name='field_landing_contents[2][subform][field_main_contents][3][subform][field_source_view][0][target_id]']").select(sourceDropdown)
+    cy.wait(1000)
```

Comment:
take this wait outside of the step and add 
And browser waits 
in the feature file